contracts/LinniaUsers.sol

```diff
@@ -44,20 +45,31 @@ contract LinniaUsers is Ownable, Pausable, Destructible {
         return true;
     }
 
-    // setProvenance allows admin to set the provenance of a user
-    function setProvenance(address user, uint provenance)
+     // setExpertScore allows admin to set the expert score of a user from a trusted third party
+    function setExpertScore(LinniaWhitelistI whitelist, address user)
         onlyOwner
```

Comment:
I don't like the idea of only us being able to operate whitelists. I think this `onlyOwner` modifier should be moved to the `updateScore` interface. Therefore only the creator of a whitelist can set scores, but users of the protocol could come with their own whitelists

Reply:
I had originated with setting onlyOwner in the updateScore interface but got compilation warning:

```
Compilation warnings encountered:

/Users/anujshah/Desktop/Work/Linnia-Smart-Contracts/contracts/LinniaWhitelistI.sol:23:5: Warning: Modifiers of functions without implementation are ignored.
    function updateScore(address user, uint score) public onlyOwner returns(bool);
    ^----------------------------------------------------------------------------^
```
The mock I made includes the modifier. 

The users of the protocol can then use the interface and create and operate whitelists however they wish. The only thing we control is bringing scores of experts on whitelists onto the main users contract. Also currently we use setProvenance within setExpertScore and that is onlyOwner as well. Let me know if that makes sense.

Reply:
onlyOwner makes sense here since we are calling the setProvenance method.  

Also if anyone could change provenance, then provenance would be useless.